nextjs-template/src/services/notehub/AppEvents.ts

```diff
@@ -18,6 +23,23 @@ export function locationAlternativesFromRoutedEvent(
   event: NotehubRoutedEventLocationFields
 ): NotehubLocationAlternatives {
   const alternatives: NotehubLocationAlternatives = {};
+  if (
+    event.best_location_when &&
+    event.best_location &&
+    event.best_country &&
+    event.best_timezone &&
+    event.best_lat &&
+    event.best_lon
```

Comment:
Is including all 6 fields in the `if` to TypeScript happy? Presumably if lat/lon are here the others should be available as well.

Reply:
Yes, it appears to be here explicitly to make TS happy